app/models/dish.rb

```diff
@@ -18,16 +18,24 @@ class Dish < ActiveRecord::Base
   RATE_DIMENSION_DISH = 'dish'
   ratyrate_rateable RATE_DIMENSION_DISH
 
-  def rates
+  # public activity
+  include PublicActivity::Model
+  tracked owner: Proc.new{ |controller, model| controller.try(:current_user) }, recipient: Proc.new{ |controller, model| model.try(:user) } 
```

Comment:
could you give me an example regarding moving Proc logic to a method? thanks! I might need to revisit Proc concepts.